materialized-views/dbt_project.yml

```diff
@@ -3,10 +3,10 @@ version: '0.2.0'
 config-version: 2
 require-dbt-version: ">=0.18.0"
```

Comment:
Given that the new configs are only supported in v1.0+, could you bump this as well?
```
require-dbt-version: ">=1.0.0"
```

Reply:
Ah that is a very relevant change. I've implemented this.